.github/workflows/build_quay.yml

```diff
@@ -0,0 +1,27 @@
+name: "Build Quay image"
+on:
+  push:
+    branches:
+      - 'stable*'
+
+jobs:
+  quay_build:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ secrets.QUAY_URI }}
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      -
+        name: Build and push
+        id: quay_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: quay.io/ocsci/ocs-ci:latest
```

Comment:
We will likely also want to tag a container image using git tag of coresponding stable build (eg. `ee also stable-ocs-4.7-202102180854`). See https://github.com/red-hat-storage/ocs-ci/tags